sycl/include/CL/sycl/handler.hpp

```diff
@@ -386,7 +386,11 @@ class __SYCL_EXPORT handler {
                        static_cast<int>(AccessTarget), ArgIndex);
   }
 
-  template <typename T> void setArgHelper(int ArgIndex, T &&Arg) {
+  template <typename T>
+  typename std::enable_if<std::is_trivially_copyable<T>::value &&
+                              std::is_standard_layout<T>::value,
```

Comment:
```suggestion
                              std::is_standard_layout<T>::value,
```
I think this requirement was lifted in SYCL-2020. I suggest enforce it only for SYCL-1.2.1 version and keep `is_trivially_copyable` only otherwise.

Reply:
Done